### ISSUE TITLE
Incorrect Type in package.json, Breaks Webpack 5

### ISSUE THREAD

**Issue by thequailman** [commonmark/commonmark.js#201]
Hello, this repo uses the wrong type specified in the package.json:

https://github.com/commonmark/commonmark.js/blob/master/package.json#L18

Webpack thinks it's commonjs, but it uses import/export syntax and breaks the loader.  See this issue for a similar repo with the same problem and a workaround until it's fixed: https://github.com/webpack/webpack/issues/11597

**Comment by jgm** [comment]
The intent was to preserve backwards-compatibility, so the package can still be used as commonjs.
If you've got a better way to do that, let us know. I don't know much about the JS ecosystem, and the person who did the PR for modules has been unresponsive.

**Comment by jgm** [comment]
As you can see from `bin/commonmark`, you can use  ESM.

**Comment by jgm** [comment]
See #195 -- is this a dup or a distinct issue?

**Comment by thequailman** [comment]
I think if you're using import/export syntax in `/lib`, you have to set the `package.json` `"type":` to `"module"`.  I fixed the issue by adding this rule:

```
			{
			  test: /\.js/,
			  include: /commonmark[\\/]lib[\\/]/,
			  type: "javascript/auto"
			},
```

**Comment by jgm** [comment]
This should be fixed by #203
